__tests__/synonyms.server.ts

```diff
@@ -10,6 +17,12 @@ import {
 } from '~/routes/synonyms/synonyms.server'
 
 jest.mock('@architect/functions')
+jest.mock('github-slugger', () => ({
+  slug: (eventId: string) => {
+    return eventId.toLowerCase()
+  },
+}))
+
```

Comment:
Would you please add a slug to explain why you are mocking github-slugger?

Reply:
I added a comment above each mock. Let me know if you wanted something different.